### DISCUSSION TITLE
Website on localhost is not working

### DISCUSSION THREAD

**Discussion by dev-arminder** [sButtons/sbuttons#1570] [Q&A]
Hi Guys, I was setting up repo on my local machine and when I run command `npm start` in website directory, an error comes into command line.
![image](https://user-images.githubusercontent.com/85746323/141610057-b2829bdf-f1c2-4d98-9391-255ea7c435b7.png)

I would be great if anyone have any idea on how to resolve it or any resources from where I can get help.

**Comment by shahednasser** [comment]
Hi @dev-arminder sorry for the delay in the reply and thank you for submitting this. Can you let me know what steps did you take to setup the project? Also, what Node version are you using?
